dmscripts/export_dos_opportunities.py

```diff
@@ -42,7 +42,7 @@ def _build_row(brief, brief_responses):
         PUBLIC_BRIEF_URL.format(brief['id']),
         brief['frameworkSlug'],
         brief['lotSlug'],
-        brief.get('specialist', ""),
+        brief.get('specialistRole', ""),
```

Comment:
It might be worth updating the column heading above to also read "Specialist Role"

Reply:
@lfdebrux I don't mind doing this, but my thought was that if people have scripts which rely on the existing field names they would break if we rename an existing field.

Reply:
That's a good point. On the other hand their scripts can't have been doing much with a field that is always empty... Not sure. Up to you!

Reply:
I guess that scripts may allow for empty value though, whereas they might trip/break on a different field name, so my preference is to keep it as-is.